docs/reference/data-streams/change-mappings-and-settings.asciidoc

```diff
@@ -69,6 +69,8 @@ PUT /_data_stream/my-data-stream
 POST /my-data-stream/_rollover/
 
 PUT /_data_stream/new-data-stream
+
+DELETE /_data_stream/*/_lifecycle
```

Comment:
Because lifecycle is still in preview I do not want to change the documentation to include the default lifecycle. This is why I chose to explicitly delete the lifecycle. When/if the data stream lifecycle becomes GA we can update the docs and remove this.